src/boot/logger.ts

```diff
@@ -0,0 +1,34 @@
+import winston from 'winston';
+import 'setimmediate';
+
+const logger = winston.createLogger({
+	level: 'info',
+	format: winston.format.simple(),
+	defaultMeta: {
+		get location() { return window.location.href; },
+		userAgent: window.navigator.userAgent,
+		service: 'webwork3'
+	},
```

Comment:
Is there a way to change this so that this meta data is only logged for the http transport. This is useless information for the developer build in the console, and makes the messages rather inflated.

Reply:
Yep -- I've simplified the console logs to avoid the default metadata